src/Avalonia.Controls.ColorPicker/ColorView/ColorView.Properties.cs

```diff
@@ -0,0 +1,495 @@
+                359);
+
+        /// <summary>
+        /// Defines the <see cref="MaxSaturation"/> property.
+        /// </summary>
+        public static readonly StyledProperty<int> MaxSaturationProperty =
+            AvaloniaProperty.Register<ColorView, int>(
+                nameof(MaxSaturation),
+                100);
+
+        /// <summary>
+        /// Defines the <see cref="MaxValue"/> property.
+        /// </summary>
+        public static readonly StyledProperty<int> MaxValueProperty =
+            AvaloniaProperty.Register<ColorView, int>(
+                nameof(MaxValue),
+                100);
+
+        /// <summary>
+        /// Defines the <see cref="MinHue"/> property.
+        /// </summary>
+        public static readonly StyledProperty<int> MinHueProperty =
+            AvaloniaProperty.Register<ColorView, int>(
+                nameof(MinHue),
+                0);
+
+        /// <summary>
+        /// Defines the <see cref="MinSaturation"/> property.
+        /// </summary>
+        public static readonly StyledProperty<int> MinSaturationProperty =
+            AvaloniaProperty.Register<ColorView, int>(
+                nameof(MinSaturation),
+                0);
+
+        /// <summary>
+        /// Defines the <see cref="MinValue"/> property.
+        /// </summary>
+        public static readonly StyledProperty<int> MinValueProperty =
+            AvaloniaProperty.Register<ColorView, int>(
+                nameof(MinValue),
+                0);
+
+        /// <summary>
+        /// Defines the <see cref="PaletteColors"/> property.
+        /// </summary>
+        public static readonly StyledProperty<IEnumerable<Color>?> PaletteColorsProperty =
+            AvaloniaProperty.Register<ColorView, IEnumerable<Color>?>(
+                nameof(PaletteColors),
+                null);
+
+        /// <summary>
+        /// Defines the <see cref="PaletteColumnCount"/> property.
+        /// </summary>
+        public static readonly StyledProperty<int> PaletteColumnCountProperty =
+            AvaloniaProperty.Register<ColorView, int>(
+                nameof(PaletteColumnCount),
+                4);
+
+        /// <summary>
+        /// Defines the <see cref="Palette"/> property.
+        /// </summary>
+        public static readonly StyledProperty<IColorPalette?> PaletteProperty =
+            AvaloniaProperty.Register<ColorView, IColorPalette?>(
+                nameof(Palette),
+                null);
+
+        /// <summary>
+        /// Defines the <see cref="SelectedIndex"/> property.
+        /// </summary>
+        public static readonly StyledProperty<int> SelectedIndexProperty =
+            AvaloniaProperty.Register<ColorView, int>(
+                nameof(SelectedIndex),
+                (int)ColorViewTab.Spectrum);
+
+        /// <summary>
+        /// Defines the <see cref="ShowAccentColors"/> property.
+        /// </summary>
+        public static readonly StyledProperty<bool> ShowAccentColorsProperty =
```

Comment:
Note: All other bool properties here use "Is" as the prefix. "Show" should be reconsidered here although it is common as an alternative in .NET and does shorten length and make more readable in cases like this.

Reply:
Tentatively renamed to `sAccentColorsVisible`. Full overview will be in a comment.